.gitignore

```diff
@@ -1 +1,2 @@
-/.vscode/
```

Comment:
Is this important to keep .vscode fine in the main project?

Reply:
Yes, it contains config such as auto formatting on save and things like that.

Feel free to merge on master when you're ready